test/unit/openmrs.spec.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+/* jasmine specs for openmrsRest service go here */
+describe('OpenmrsRest service', function() {
+
+    beforeEach(function(){
+        jasmine.addMatchers({
+            toEqualData: function(util, customEqualityTesters) {
+                return {
+                    compare: function(actual, expected) {
+                        var passed = angular.equals(actual, expected);
+                        return {
+                            pass: passed,
+                            message: 'Expected ' + actual + (passed ? '' : ' not') + ' to equal ' + expected
+                        }
+                    }
+                }
+            }
+        });
+    });
+
+    beforeEach(module('conceptDictionaryApp'));
+
+    describe('ListFull should retrun full representations', function(){
+        var $httpBackend, openmrsRest, list;
+
+        beforeEach(inject(function(_$httpBackend_, _openmrsRest_){
+        	openmrsRest = _openmrsRest_;
+            $httpBackend = _$httpBackend_;
+            $httpBackend.whenGET('partials/index-menu.html').respond();
+            $httpBackend.whenGET('/ws/rest/v1/item?v=full').
+            respond({results:[{ name: 'item1',  
+            					uuid: "2b22dc27-72ec-4ab5-9fa8-d98be91adc1c"},
+            				  { name: 'item2', 
+            					uuid: "3g65dc27-72ec-4ab5-9fa8-d98be91adb5b"},
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2319554381/issues/source?bid=3196969&fileBranchId=3204313#l35)